metron-platform/metron-parsers/src/test/java/org/apache/metron/writers/integration/WriterBoltIntegrationTest.java

```diff
@@ -88,9 +108,7 @@ public void initialize(Map<String, Object> validationConfig, Map<String, Object>
   public static String parserConfigJSON;
 
   @Test
-  public void test() throws UnableToStartException, IOException, ParseException {
-
-    UnitTestHelper.setLog4jLevel(CSVParser.class, org.apache.log4j.Level.FATAL);
+  public void parser_with_global_validations_writes_bad_records_to_error_topic() throws Exception {
```

Comment:
Maybe just a brief javadoc would help here.  I have to read a lot of code to understand what each of these tests are trying to do. One sentence of javadoc would save me some time.

Reply:
I think the quantity of code is mainly due to the existing setup code. I'm happy to refactor that. I really don't like adding or reading javadoc where code can communicate clearly. I think it's redundant to add it where method and class names can appropriately capture the intent. With extracting out the common setup code and the Processor logic, does the test name you've referenced on line 111 along with the test assertions make it clear enough?

Reply:
> I really don't like adding or reading javadoc where code can communicate clearly. I think it's redundant to add it where method and class names can appropriately capture the intent. 

Yes, I can see your point, but the definition of what is clear can be very different between the original author and others maintaining that code.  Or even the original author and their future self, months later. Redundancy is a small price to pay for clarity. This is especially true in an open source project which we hope long outlives ourselves.  But we can disagree on this point.

I offered the refactor/rename comments only as suggestions.  Do whatever you think is sufficient to make it more clear.  I'm sure any little effort to improve clarity will win me over.  I'll be exceptionally pragmatic here.